ios/src/ImageCropPicker.m

```diff
@@ -434,6 +434,10 @@ - (void) handleVideo:(AVAsset*)asset withFileName:(NSString*)fileName withLocalI
             [outputURL getResourceValue:&fileSizeValue
                                  forKey:NSURLFileSizeKey
                                   error:nil];
+            
+            AVURLAsset *durationFromUrl = [AVURLAsset assetWithURL:outputURL];
+            CMTime time = [durationFromUrl duration];
+            int seconds = ceil(time.value/time.timescale) * 1000;
```

Comment:
I guess name of variable here should be milliseconds?

Reply:
oh, yes, you are right

Reply:
Changed